CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project are documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [v1.3.0] 2020-02-14
+
+### Added
+
+- **CUMULUS-1896**
+  - Added a 'streaming' mode to the CMA.   This will allow client libraries to avoid requiring three subprocess/python invocations when running the CMA commands in sequence, thus lowering
+    the overhead of utilizing the CMA.
+
+### BREAKING CHANGES
+
+- **CUMULUS-1896**
+  - Updated CMA precompiled binary files to a layer subdirectory `./cma_bin`. Users who utilize the pre-compiled binary should update their pathing expectations to that directory.   Users utilizing the distribution zip file as a layer should update their CMA executable target to `/opt/cma` on AWS.
```

Comment:
I don't grok what this means:

> Users who utilize the pre-compiled binary should update their pathing expectations to that directory

Is it the same thing as the next line about updating the CMA executable target?

Also, is there any other way we can better advertise these kinds of differences in the Cumulus core docs? As someone who recently emerged from a fun round of CMA debugging, I worry about a note like this getting buried in release docs. I wonder if the Cumulus docs could at least specify the different value of `CUMULUS_MESSAGE_ADAPTER_DIR` for different CMA releases

Reply:
Remove "Users utilizing the distribution zip file as a layer should update their CMA executable target to `/opt/cma` on AWS."

Reply:
re: your comment, yeah, I think we need to add a doc PR in addition to CHANGELOG commentary in Core for core users.

Reply:
f1272be